src/sdk/endpoint_stats_collector.js

```diff
@@ -218,8 +227,9 @@ class EndpointStatsCollector {
 
     update_nsfs_read_counters({ size = 0, count = 0, is_err }) {
         if (is_err) {
-            this.nsfs_io_counters.error_read_count += count;
-            this.nsfs_io_counters.error_read_bytes += size;
+            dbg.warn(`unexpectedly reached here upon error, we need to figure out why and maybe re-add the error counters`);
+            // this.nsfs_io_counters.error_read_count += count;
```

Comment:
I think that we should not leave code in comments in general.
If we would get here and need to debug it - we have version control.

If you insist on leaving them, at least I would suggest also leaving a comment here about why we added the code in a comment explicitly. For example:
```
Uncomment these lines in case we see the warning above in the logs.
I left them to help us debug it easily.
We can delete them if we will see that we don't have the warning above for a while.
```

Reply:
The warning itself is the explanation.
here we will probably never reach the error path as it is in under layer, and the error will fail much earlier.
with that said, this code is relaying on an older code, and for the chance that we are missing a flow, we added the warning to explain and to see that we actually moved throw this flow.

once we get confidante we can remove  the condition completely (the whole if, else will not be relevant).